File: Assignment36_1.c
#include <stdio.h>

///////////////////////////////////////////////////////////////////////////////
// Function Name : strlwrx
// Description   : Convert complete string to lowercase
// Input         : String
// Output        : Modified string in lowercase
// Author        : Raj Samir Jadhav
// Date          : 07/12/2025
///////////////////////////////////////////////////////////////////////////////

void strlwrx(char *str)
{
    while(*str != '\0')                     // Traverse string till end
    {
        if((*str >= 'A') && (*str <= 'Z'))  // Check for capital letters
        {
            *str = *str + 32;               // Convert to lowercase (ASCII logic)
        }
        str++;                              // Move to next character
    }
}

int main()
{
    char arr[30];

    printf("Enter string : ");
    scanf("%[^'\n']s", arr);

    strlwrx(arr);                            // Function Call

    printf("Modified string is : %s\n", arr);

    return 0;
}

File: Assignment36_1.java
import java.util.Scanner;

///////////////////////////////////////////////////////////////////////////////
// Function Name : strlwrx
// Description   : Convert complete string to lowercase
// Input         : String
// Output        : Modified string in lowercase
// Author        : Raj Samir Jadhav
// Date          : 07/12/2025
///////////////////////////////////////////////////////////////////////////////
class Pattern
{
    public String strlwrx(String str)    // Function returns modified string
    {
        String result = "";

        for(int i = 0; i < str.length(); i++)
        {
            char ch = str.charAt(i);

            if(ch >= 'A' && ch <= 'Z')   // Capital to lowercase
            {
                ch = (char)(ch + 32);
            }

            result = result + ch;        // Store updated char
        }
        return result;
    }
}

class Assignment36_1
{
    public static void main(String[] args)
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter string : ");
        String input = sobj.nextLine();

        Pattern pobj = new Pattern();
        String output = pobj.strlwrx(input);

        System.out.println("Modified string is : " + output);

        sobj.close();
    }
}
